install_gtc_generation_minimal_dependencies.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Install htslib
+htsversion=1.3
+sudo curl -L https://github.com/samtools/htslib/releases/download/${htsversion}/htslib-${htsversion}.tar.bz2 | tar xj && \
+    (cd htslib-${htsversion} && /
+    sudo ./configure --enable-plugins --with-plugin-path='$(libexecdir)/htslib:/usr/libexec/htslib' && /
+    sudo make install)
+
+# Install Illumina iaap-cli
+cd $HOME && \
+    aws s3 cp s3://scratch-embark/iaap/iaap-cli-linux-x64-1.1.0-sha.80d7e5b3d9c1fdfc2e99b472a90652fd3848bbc7.tar.gz . && \
+    tar -xf $HOME/iaap-cli-linux-x64-1.1.0-sha.80d7e5b3d9c1fdfc2e99b472a90652fd3848bbc7.tar.gz && \
+    chmod +x $HOME/iaap-cli-linux-x64-1.1.0-sha.80d7e5b3d9c1fdfc2e99b472a90652fd3848bbc7/iaap-cli/iaap-cli && \
+    rm $HOME/iaap-cli-linux-x64-1.1.0-sha.80d7e5b3d9c1fdfc2e99b472a90652fd3848bbc7.tar.gz
+
+export PATH="$HOME/iaap-cli-linux-x64-1.1.0-sha.80d7e5b3d9c1fdfc2e99b472a90652fd3848bbc7/iaap-cli:$PATH"
+
+# Install BeadArrayFiles library for Illumina file manipulation
+cd $HOME
+sudo apt-get update -y
+sudo apt-get install -y python-distutils-extra
+sudo apt-get install -y  python3-distutils
+git clone https://github.com/Illumina/BeadArrayFiles.git && \
+    cd BeadArrayFiles; python3 setup.py install
```

Comment:
Is this the same as the parts in the Dockerfile? If so, you could [COPY](https://docs.docker.com/engine/reference/builder/#copy) this in and then call this script in the Dockerfile

Reply:
I ended up keeping this separate from the Docker file because I was having trouble getting the container to run on an instance (the container runs fine from my local machine, but I needed more RAM for the work I'm doing at the moment)

Reply:
I think I'd rather put that off until we have a better idea of how/when/by whom this code is going to be used.